adapters/oidc/tomcat/pom.xml

```diff
@@ -30,6 +30,18 @@
     <artifactId>keycloak-tomcat-integration-pom</artifactId>
     <packaging>pom</packaging>
 
+    <dependencyManagement>
```

Comment:
I meant Tomcat adapters should not use dependency versions from JBoss AS 7/EAP. Tomcat is an active project, while JBoss AS7 is long been replaced by WildFly.